app/assets/stylesheets/home.scss

```diff
@@ -1,9 +1,41 @@
 body {
-  background: image_url("polygon_green.jpg") no-repeat center center fixed;
+  background: image_url("gray_gradient.jpg") no-repeat center center fixed;
```

Comment:
Name of function `image_url` should be written in all lowercase letters with hyphens instead of underscores